code/drasil-printers/lib/Language/Drasil/JSON/Print.hs

```diff
@@ -27,33 +27,43 @@ import Language.Drasil.JSON.Helpers (makeMetadata, h, stripnewLine, nbformat,
  tr, td, image, li, pa, ba, table, refwrap, refID, reflink, reflinkURI, mkDiv)
 
 -- | Generate a python notebook document (using json).
-genJSON :: PrintingInformation -> L.Document -> Doc
-genJSON sm doc = build (makeDocument sm doc)
+-- build : build the general Jupyter Notbook document
+-- build': build the SRS example in JSON format
+genJSON :: PrintingInformation -> String -> L.Document -> Doc
+genJSON sm "Jupyter" doc = build (makeDocument sm doc)
```

Comment:
Matching on a `String` is a bad idea. Create a data type (whose contents can be, for example `Jupyter` and `Other`).

Reply:
We have a ``DocType`` data in drasil-gen but it's not a dependency of the printer (drasil-printer is a dependency of drasil-gen though). Should we move the ``DocType`` to printer so we can use the information in both places?

Reply:
That's likely not going to be the final resting place for `DocType`, but this will work for now.